Formula/httest.rb

```diff
@@ -18,15 +18,15 @@ class Httest < Formula
   depends_on "lua"
 
   def install
+    (buildpath/"brew_include").install_symlink Formula["pcre"].opt_include => "pcre"
+    ENV.prepend "CPPFLAGS", "-I#{buildpath}/brew_include"
```

Comment:
The include is broken because it expects the files to be in a subdirectory named pcre.

Locally at least, it is indeed linked so not sure what you mean.